canvas/image.go

```diff
@@ -79,22 +108,97 @@ func (i *Image) Resize(s fyne.Size) {
 	}
 
 	i.baseObject.Resize(s)
+	i.updated = false
+	i.Refresh()
+}
 
-	Refresh(i)
+// Aspect will return the original content aspect after it was last Refresh
+func (i *Image) Aspect() float32 {
```

Comment:
This is necessary for the painter to be able to properly scale the Image and can not be guessed from it as the SVG resize might lead to a different aspect.

Reply:
Why would resizing change the aspect of the image source?